tools/download-test.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+while [[ $# -gt 1 ]]
+do
+  key="$1"
+
+  case $key in
+      -t|--tap)
+      TAP="$2"
+      shift # past argument
+      ;;
+      *) # unknown option
+      ;;
+  esac
+  shift # past argument or value
+done
+
+if [ -z ${NODE_VERSION} ]; then
+  echo "NODE_VERSION is undefined! Please declare NODE_VERSION"
+  exit 1
+fi
+# Remove old files
+[ ! -z ${TAP} ]; rm -r ${TAP} || true
+#rm -r nodejs.org || true
```

Comment:
believe this needs to be -rf to avoid failures if the directory does not already exist.